src/util/system.cpp

```diff
@@ -388,7 +388,7 @@ bool ArgsManager::ParseParameters(int argc, const char* const argv[], std::strin
             key.erase(is_index);
         }
 #ifdef WIN32
-        std::transform(key.begin(), key.end(), key.begin(), ToLower);
```

Comment:
This seems to clearly perform worse. Is there a reason not to keep the current way? (A std::string ToLower overload could be *in addition* to this.)

Reply:
@luke-jr this bit is moved to #16566

Reply:
I could keep the in-place transform, but seriously, this is called a couple of dozen times once at startup and not again. I think we want to aim for readability on this one.